### ISSUE TITLE
need for game citation format

### ISSUE THREAD

**Issue by snapdragoness** [sigchi/Document-Formats#45]
Those working in the game space often need to cite games in our papers; authors have taken an ad hoc approach to this so far. I would like to move this forward into the official templates so that we have a clear format for games.

The _Game Studies_ journal has formal specifications for citing games (http://gamestudies.org/1501/submission_guidelines#GSCitation), which are based on APA format. I propose using the _Game Studies_ journal format as a basis, but adapt it to match the format that we are using in SIGCHI. This puts it somewhere between book format and the various misc formats (online video, tweets, etc.). The style needs to have the game title be italicized, must include the publisher information, and should have the "`Game [<platform>]`" tag (similar to the Tweet and Online Video samples). 

Here is a proposed format, which references a particular instance of a game (release date, platform): 

``` xml
<Developer>. <Year Released>. <Game Title (italicized, like a book)>. Game [<platform played / analyzed for manuscript>]. <Publisher>, <Release Location>. Played <most recent play date>.
```

Here are examples:

Atari, Inc. 1979. _Adventure_. Game [Atari 2600]. Atari, Inc., Sunnyvale, CA, USA. Played August 25, 2007.

343 Industries. 2012. _Halo 4_. Game [Xbox 360]. Microsoft Studios, Redmond, WA, USA. Played December 31, 2012.

Namco. 2004. _Katamari Damacy_. Game [PS2]. Namco, Tokyo, Japan. Played January 20, 2005.

Lucas Pope. 2013. _Papers, Please_. Game [OS X]. Self-published. 

I surfaced this on the CHI Games Facebook group, to little interest. Unclear if there is another body that might have a helpful opinion.

**Comment by ayman** [comment]
Can you test various BibTeX entries and show an example that render correctly?

**Comment by snapdragoness** [comment]
It is a bit of a hack, but it seems the `inproceedings` type will enable this, as long as the game title is marked with `\emph`. Two bib entries follow; sample using CHI 2016 release attached.

``` TeX
@inproceedings{atari:adventure,
    Address = {Sunnyvale, CA, USA},
    Author = {{Atari, Inc.}},
    Howpublished = {Game [Atari 2600]},
    Month = {December},
    Note = {Played August 25, 2007},
    Publisher = {{Atari, Inc.}},
    Title = {\emph{Adventure}},
    Year = {1979}}
```

``` TeX
@inproceedings{343Industries:2012,
    Address = {Redmond, WA, USA},
    Author = {{343 Industries}},
    Howpublished = {Game [Xbox 360]},
    Month = {Nov},
    Note = {Played December 30, 2012.},
    Publisher = {Microsoft Studios},
    Title = {\emph{Halo 4}},
    Year = {2012}}
```

![screen shot 2015-09-09 at 11 10 30 am](https://cloud.githubusercontent.com/assets/3452924/9768838/8e6af8f6-56e3-11e5-8761-c3912904501d.png)

**Comment by snapdragoness** [comment]
Also, I tested with almost all bib types. `misc` would work if we changed the format to include publisher information.

**Comment by ayman** [comment]
Please provide `misc` example as it's likely more apropos than `inproceedings`.

**Comment by snapdragoness** [comment]
Here is one of them as a `misc` example; yes, this semantically makes much more sense. The `.bst` file does not put the publisher and publisher's address into the citation. I spent some time trying to fix this in my environment, but could not make it work. Bib data follows, as well as screenshot of typeset output.

``` TeX
@misc{343Industries:2012,
    Address = {Redmond, WA, USA},
    Author = {{343 Industries}},
    Howpublished = {Game [Xbox 360]},
    Month = {Nov},
    Note = {Played December 30, 2012.},
    Publisher = {Microsoft Studios},
    Title = {\emph{Halo 4}},
    Year = {2012}}
```

<img width="376" alt="screen shot 2015-09-13 at 11 03 31 pm" src="https://cloud.githubusercontent.com/assets/3452924/9842474/d11cdc92-5a6b-11e5-9c58-fdedfb42a2ab.png">

**Comment by ayman** [comment]
I'd rather not add a `@game` type if we don't have to.  And sorry to keep going back and forth @toupsz - do you think ALL of these fields are needed in a game ref? I guess what i'm asking is broadly, what are other communities attempting here beyond the one you cited at the start. Alternatively, we might could edit the misc format for SIGCHI to push the extra fields. But I'd want to pull in some other experts.

**Comment by ayman** [comment]
I guess what i'm asking is broadly, what are other communities attempting here beyond the one you cited at the start.

**Comment by snapdragoness** [comment]
I've googled around for a few other communities; not finding a lot of data. They do all appear to include publisher information. At the same time, I am unsure that it is really critical to track down a particular instance of a cited game. If you have the developer, title, and release date, that is likely sufficient.

I don't think we need an `@game` type. I guess I was unclear earlier; could we not just provide publisher and address on the `@misc` type (and if it is missing, nothing happens)? That is what I was working on when hacking on the .bst file (but I lack sufficient brainpower when I've had time to work on it of late). If that is too much work, I don't think it would hurt to just cite games without publisher data.

**Comment by SimonMcCallum** [comment]
I believe that we should have an @game type.  Not from the pragmatic point of view where using misc is obviously better, but from a recognition perspective.  The ACM recognising that game are a valid form of published material is an important part of having an academic community become more accepting of the form.  I have a .bst with alterations to both @misc and @game which I will upload after dinner

**Comment by ayman** [comment]
The problem with making a `@game` format is it is specific to just our `bst` - so you flip from `sigchi` to `apa` format for another conference and BibTex just errors not knowing what a `@game` is.

**Comment by snapdragoness** [comment]
I think it is sufficient to have a proper citation format for games, without the need to add a new type. I agree with @ayman that an `@game` type will break compatibility of your bib file with other formats. Further, I suspect it's a lot of dev work with a relatively small payoff when we can use `@misc` with some minor modifications.

**Comment by SimonMcCallum** [comment]
Here is my bst misc

```
FUNCTION {misc}
{ output.bibitem
  format.authors output
  new.block
  format.date output
  title howpublished new.block.checkb
  title empty$ 'skip$ 'possibly.setup.inlinelink if$ % urlbst
  format.title output
  howpublished new.block.checka
  howpublished output
  publisher empty$ 
    'skip$ 
    {publisher "publisher" output.check}
  if$
  address empty$ 
    'skip$ 
    {address "address" output.check}
  if$
  new.block
  note output
  fin.entry
  empty.misc.check
}
```

**Comment by snapdragoness** [comment]
@SimonMcCallum I tried this but could not get it to work with my bibliographies (issues with popping an empty stack). Does it work in your environment for all of the `@misc` examples?

**Comment by snapdragoness** [comment]
So it seems like the remaining issue is still that we can't get publisher and publisher address to appear for games; is that right?

**Comment by ayman** [comment]
Can you come up with any suggestion that might work using the current `@misc`?

**Comment by snapdragoness** [comment]
The only solution I have presently is to put the extra information into the Note field:

`2K Games, Novato California, USA. Last played January 2009.`

That seems like it's overloading things a bit, but we can make it work. Is it unreasonable to simply add publisher and address to the .bst for `@misc` like @SimonMcCallum did (assuming we can get it to work)?

[As an aside, this is coming back up because we are about to freeze the template for CHI PLAY 2016 and I would like to make sure there is a way for authors to cite games easily.]

**Comment by ayman** [comment]
So...I'm trying very very hard not to try to change that bst (unless of course the ACM will take the edits back)...bc then we have to manually track and merge when they make updates.  So from a long term maintenance point of view, its not desirable.   What's your deadline for freezing chi play's template?

**Comment by snapdragoness** [comment]
Papers are due April 18; I'd say no later than Feb 18 (so maybe much too soon). I see the problem with the bst file now; do we have a mechanism to push changes back to the ACM? Maybe we shelve for CHI PLAY this year and aim for next?

**Comment by ayman** [comment]
Here's my suggestion.  We make a sample entry using misc and deal for now. We'll see if the ACM might take our BST edit...if so, we'll make a fork and go from there.

**Comment by snapdragoness** [comment]
Sounds good.

**Comment by snapdragoness** [comment]
@ayman should I fork and add the sample entries then send a pull request? (Then we can freeze a copy after that for the conference?) Can we get new builds after that? (Or should I make them part of the fork?)

**Comment by ayman** [comment]
Ya.  Add it to all 4 templates and send me a PR. I'll take the PR and generate builds myself.   But if you wanna try your hand at making the builds, by all means. :-)  I forget if i documented that part.  You have to save as the two pdfs into the examples folder from word.  Then generate the LaTeX in place.  then call in the `builds/bin` dir clean, pull, then make.

**Comment by snapdragoness** [comment]
Done, including building. All steps seemed clear to me!

New PR: #72

**Comment by ayman** [comment]
Closed with PR #75.
